### PR TITLE
DOC: speed up some examples

### PR DESCRIPTION
**Pull request by mdhaber**
#### Reference issue
gh-20420

#### What does this implement/fix?
Speeds up some slow examples to reduce doc build times.

### CODE REVIEW

**Comment by mdhaber** [comment]
Oops, I missed `generic_filter`. For that, I'd also propose just applying a 0.5 zoom before processing.

**Comment by mdhaber** [comment]
To check for slow distribution methods:

<details>

```python3
import numpy as np
from scipy import stats
from scipy.stats._distr_params import distcont
rng = np.random.default_rng(2549824598234528)
from time import perf_counter

distcont = dict(distcont)

for distname, params in distcont.items():
    dist = getattr(stats, distname)
    tic = perf_counter()
    mean, var, skew, kurt = dist.stats(*params, moments='mvsk')
    toc = perf_counter()
    stats_time = toc - tic

    if stats_time > 0.1:
        print(f"{distname} stats slow: {stats_time}")

    tic = perf_counter()
    x = np.linspace(dist.ppf(0.01, *params),
                    dist.ppf(0.99, *params), 100)
    vals = dist.ppf([0.001, 0.5, 0.999], *params)
    toc = perf_counter()
    ppf_time = toc - tic

    if ppf_time > 0.1:
        print(f"{distname} ppf slow: {ppf_time}")

    tic = perf_counter()
    dist.pdf(x, *params)
    toc = perf_counter()
    pdf_time = toc - tic

    if pdf_time > 0.1:
        print(f"{distname} pdf slow: {pdf_time}")

    tic = perf_counter()
    dist.cdf(vals, *params)
    toc = perf_counter()
    cdf_time = toc - tic

    if cdf_time > 0.1:
        print(f"{distname} cdf slow: {cdf_time}")

    if distname != 'studentized_range':
        tic = perf_counter()
        dist.rvs(*params, size=1000)
        toc = perf_counter()
        rvs_time = toc - tic

    if rvs_time > 0.1:
        print(f"{distname} rvs slow: {rvs_time}")
```

</details>

<details>

```
argus stats slow: 0.3494060420198366
exponpow stats slow: 0.13523816602537408
exponweib stats slow: 0.1725937090232037
gausshyper rvs slow: 1.5470748749794438
genexpon stats slow: 0.10502708400599658
halfgennorm stats slow: 0.13357295899186283
johnsonsb stats slow: 0.13170608301879838
kappa4 stats slow: 1.0273430829984136
ksone stats slow: 3.2752658340032212
ksone rvs slow: 2.4722056669997983
kstwo stats slow: 1.527090416988358
kstwo rvs slow: 0.6692847079830244
kstwobign stats slow: 0.13993191596819088
levy_stable pdf slow: 0.12228358298307285
powerlognorm stats slow: 0.20370729197748005
powernorm stats slow: 0.17204587499145418
recipinvgauss stats slow: 2.033138416998554
rel_breitwigner rvs slow: 1.184638625010848
studentized_range stats slow: 14.220068041991908
studentized_range ppf slow: 0.8574097499949858
studentized_range pdf slow: 1.4227888329769485
vonmises stats slow: 3.168535624979995
vonmises_line stats slow: 2.315515166032128
```
</details>

The slowest, `studentized_range.rvs`, is not actually run in the doc build.

**Comment by mdhaber** [comment]
Yeah, `rvs` and `ppf` are very slow in this infrastructure when it's numerically inverting the CDF, especially if the (CDF itself is computed using quadrature). I will override the docs for those distributions.

**Comment by mdhaber** [comment]
If CI looks good after this last commit, I think this is good to go.

File: scipy/stats/_continuous_distns.py [hunk 1]
```diff
@@ -134,7 +134,37 @@ class ksone_gen(rv_continuous):
...
+
+    Alternatively, the distribution object can be called (as a function)
+    to fix the shape, location and scale parameters. This returns a "frozen"
+    RV object holding the given parameters fixed.
+
+    Freeze the distribution and display the frozen ``pdf``:
+
+    >>> rv = ksone(n)
+    >>> ax.plot(x, rv.pdf(x), 'k-', lw=2, label='frozen pdf')
+    >>> x.legend(loc='best', frameon=False)
```
Comment (mdhaber) [inline scipy/stats/_continuous_distns.py:160]:
```suggestion
    >>> ax.legend(loc='best', frameon=False)
```

**Comment by mdhaber** [comment]
Fixed!

**Review by rgommers — APPROVED** [review]
Thanks @mdhaber! Both the CircleCI log and a local doc build indicate that the build is now measurably faster. This all looks good to me; the `correlate2d` image becoming grainy is something that we'll just accept here I think, it's a very minor thing and the build time gain is more important.

**Pull request merged**